types/office-js-preview/index.d.ts

```diff
@@ -11946,6 +11948,14 @@ declare namespace Office {
          *
          * **Important**: In Outlook on the web, `setSignatureAsync` only works on messages.
          *
+         * **Important**: If your add-in implements the 
+         * {@link https://docs.microsoft.com/office/dev/add-ins/outlook/autolaunch | event-based activation feature using `LaunchEvent` in the manifest},
+         * and calls `setSignatureAsync` in the event handler, the following behaviors apply.
+         *
+         * - If the user is editing a meeting, the `setSignatureAsync` call returns an error.
+         * - When the user composes a new item (including reply or forward), the signature is set but doesn't modify the form. This means
+         * if the user closes the form without making other edits, they won't be prompted to save changes.
```

Comment:
Maybe it is worth mentioning that a draft is also not created?

Reply:
@emtencza Will do. Thx.

Reply:
Updated